src/main/java/org/hisp/dhis/rules/models/RuleEvaluationResult.java

```diff
@@ -0,0 +1,40 @@
+package org.hisp.dhis.rules.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RuleEvaluationResult
+{
+
+    private Rule rule;
+
+    private List<RuleEffect> ruleEffects;
+
+    private boolean isEvaluated;
```

Comment:
All seems good. 
Just a small observation.
can we use `evaluatedAs` instead of `isEvaluated` ?
`isEvaluated` seems whether this rule was evaluated or not.
whereas `evaluatedAs` seems result of the evaluation is either true or false

Reply:
Nice one, I will change it